child_compassion/mappings/project_reservation_mapping.py

```diff
@@ -21,7 +21,7 @@ class ProjetReservationCreateMapping(OnrampMapping):
     MAPPING_NAME = "create_reservation"
 
     CONNECT_MAPPING = {
-        'Beneficiary_GlobalID': None,
+        'Beneficiary_GlobalID': 'global_id',
```

Comment:
Sorry you can ignore my previous comment, but you can change to `'child_id.global_id'` so that you can link the reservation to the child.